EIPS/eip-5069.md

```diff
@@ -1,44 +1,118 @@
+
+Documenting all of the things we would not do is impossible, and the above are just a few examples. We reserve the right to do less work whenever possible!
+
+## Structure
+
+### EIP Editors
+
+We, the Editors, consist of some number of EIP Editors and one Janitor elected by and from the EIP Editors.
+
+EIP Editors are responsible for governing the EIP process itself, electing a Janitor, and stewarding proposals.
+
+The Janitor's two responsibilities (above their EIP Editor duties) are: to determine when rough consensus has been reached on a matter, and determine when/if it is appropriate to re-open an already settled matter.
+
+## Membership
+
+Anyone may apply to join as an EIP Editor. Specific eligibility requirements are left to individual current EIP Editors, but the general requirements are:
+
+ - A strong belief in the above mission;
+ - Proficiency with English (both written and spoken);
+ - Reading and critiquing EIPs;
+ - Participation in governance.
+
+EIP Editors are expected to meet these requirements throughout their tenure, and not doing so is grounds for removal. Any member may delegate some or all of their responsibilities/powers to tools and/or to other people.
+
+## Making Decisions
+
+### Informally
+
+For decisions that are unlikely to be controversial&mdash;especially for decisions affecting a single proposal&mdash;an EIP Editor may choose whatever option they deem appropriate in accordance with our mission.
+
+### Formally
+
+Electing a Janitor, adding/removing EIP Editors, and any possibly-controversial decisions must all be made using variations of this formal process.
+
+#### Preparation
+
+##### Call for Input
+
+For any matter requiring a decision, a call for input must be published in writing to the usual channels frequented by EIP Editors.
+
+##### Quorum
+
+Initially, to establish a valid quorum, all EIP Editors must express their opinion (or vote where appropriate) on the matter under consideration.
+
+After thirty days from the call for input, if not all EIP Editors have responded, the quorum is reduced to the Editors that have responded. This deadline may be extended in exceptional situations.
+
+#### Deciding
+
+##### Electing a Janitor
+
+Any EIP Editor can call for an election for Janitor. The EIP Editor with the most votes once quorum is met is named Janitor until the next election completes. If there is a tie, we'll randomly choose between the EIP Editors with the most votes. Business continues as usual while the election is running.
+
+##### Adding an EIP Editor
+
+An EIP Editor is added once quorum is met, provided no current EIP Editor objects.
+
+##### Removing an EIP Editor
+
+An EIP Editor is removed once quorum is met, provided no current EIP Editor (aside from the one being removed) objects. If the removed Editor was also the Janitor, an election for a new Janitor begins immediately.
```

Comment:
```suggestion
An EIP Editor can be removed by another EIP Editor once quorum is met, provided no other EIP Editor objects. An EIP Editor can remove themself at any time.

If the Janitor ever ceases to be an EIP Editor, an election for a new Janitor begins immediately. A temporary Janitor must be appointed by the former Janitor.
```

Reply:
"can be" is the wrong word choice here. If all of the following are true:

- an Editor posts a call for input on removing an editor,
- quorum is met, and
- no one objects

Then the editor _is_ removed, not _can be_ removed.